kalman_watch/index_kalman_perigee_list.html

```diff
@@ -26,9 +26,7 @@
 </head>
 
 <body>
-    {% if prev %} <a href="{{prev}}/index.html">Previous</a>{% endif %}
-    {% if index %} <a href="{{index}}/index.html">Index</a>{% endif %}
-    {% if next %} <a href="{{next}}/index.html">Next</a>{% endif %}
```

Comment:
What is the intent here with removing Previous and Next?

Reply:
None of those things exist for this page. I think it was a leftover from copying another template.